src/store/sagas/getForecastAuto.js

```diff
@@ -1,14 +1,18 @@
 import {
   call, put, takeEvery,
 } from 'redux-saga/effects';
-import axios from 'axios';
 import * as actions from '../constants';
 
 function fetchForecastAuto({ name, key }) {
-  return axios
-    .get(`https://api.weatherbit.io/v2.0/forecast/daily?city=${name}&key=${key}`)
-    .then((res) => res.data)
-    .catch((err) => (err.response ? err.response.data.message : err.message));
+  return fetch(`https://api.weatherbit.io/v2.0/forecast/daily?city=${name}&key=${key}`)
+    .then((response) => {
+      if (response.status === 200) {
```

Comment:
There might be cases when `status` is 200, but `response.ok` is false. This issue touches this moment https://github.com/whatwg/fetch/issues/299

It's better to also check for the `ok` property.